pkg/apis/vpc/v1alpha1/ipaddressallocation_types.go

```diff
@@ -46,15 +44,15 @@ type IPAddressAllocationSpec struct {
 	// +kubebuilder:default=Private
 	// +optional
 	IPAddressBlockVisibility IPAddressVisibility `json:"ip_address_block_visibility,omitempty"`
-	// AllocationSize specifies the size of IP CIDR to be allocated.
+	// AllocationSize specifies the size of allocation_ips to be allocated.
 	AllocationSize int `json:"allocation_size,omitempty"`
 }
 
 // IPAddressAllocationStatus defines the observed state of IPAddressAllocation.
 type IPAddressAllocationStatus struct {
-	// CIDR is the allocated CIDR
-	CIDR       string      `json:"CIDR"`
-	Conditions []Condition `json:"conditions,omitempty"`
+	// allocation_ips is the allocated IP addresses
+	AllocationIPs string      `json:"allocation_ips"`
```

Comment:
I think `allocation_size` and `allocation_ips` doesn't match the naming style in Kubernetes CRD.. It should be CamelCase.

Reply:
ok